spec/hal_client/form_spec.rb

```diff
@@ -0,0 +1,113 @@
+require "hal_client/form"
+require "hal-client"
+
+RSpec.describe HalClient::Form do
+  describe ".new" do
+    specify { expect( HalClient::Form.new(fieldless_get_form, a_client) ).to behave_like_a HalClient::Form }
```

Comment:
You use `behave_like_a described_class` on https://github.com/pezra/hal-client/pull/72/files#diff-5e82c931677f94a0efe5818d1b5aa6e9R7.  Are these two equivalent?  Is one better than the other?